Soundex.test.py

```diff
@@ -4,11 +4,22 @@
 class TestSoundex(unittest.TestCase):
 
     def test_empty_string(self):
-        self.assertEqual(generate_soundex(""), "")
+        self.assertEqual(generate_soundex(""), "0000")
 
     def test_single_character(self):
         self.assertEqual(generate_soundex("A"), "A000")
 
-    
+    def testcases(self):
```

Comment:
Instead of comments better wrap each test code in to another function and comment could be the better name for test case